assets/src/data/model/endpoints.js

```diff
@@ -24,7 +24,7 @@ export const { collection_endpoints: endpoints = {} } = data.paths;
  */
 export const getEndpoint = ( modelName ) => {
 	assertEntityHasKey( modelName, endpoints );
-	return endpoints[ modelName ];
+	return '/wp-json' + endpoints[ modelName ];
```

Comment:
This change is needed because the new `wp.apiFetch` no longer automatically prepends `wp-json` as its wp-agnostic.